ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiCommandCompletionService.scala

```diff
@@ -49,9 +49,23 @@ private[apiserver] final class ApiCommandCompletionService private (
 
         completionsService
           .getCompletions(offset, request.applicationId, request.parties)
+          .via(logger.debugStream(completionsLoggable))
           .via(logger.logErrorsOnStream)
     }
 
+  private def completionsLoggable(response: CompletionStreamResponse): String =
+    s"Responding with completions: ${response.completions.toList
+      .map(c => singleCompletionLoggable(c.commandId, c.status.map(_.code)))}"
+
+  private def singleCompletionLoggable(
+      commandId: String,
+      statusCode: Option[Int],
+  ): Map[String, String] =
+    Map(
+      logging.commandId(commandId),
+      "statusCode" -> statusCode.map(_.toString).getOrElse(""),
```

Comment:
```suggestion
      "statusCode" -> statusCode.mkString,
```

Reply:
Good catch!

Reply:
Actually it's not possible with our current Scala compiler settings. Specifically, compilation will fail with:
```
error: [wartremover:Option2Iterable] Implicit conversion from Option to Iterable is disabled - use Option#toList instead
```
I could use `statusCode.toList.mkString` but it looks a bit strange to create a list only for `mkString`. What do you think?